CMakeTests/FindOProfile.cmake

```diff
@@ -0,0 +1,45 @@
+# - Try to find OProfile
+# Once done this will define
+# OPROFILE_FOUND - System has OProfile
+# OPROFILE_INCLUDE_DIRS - The OProfile include directories
+# OPROFILE_LIBRARIES - The libraries needed to use OProfile
+
+if(CMAKE_SIZEOF_VOID_P EQUAL 8)
+    set(_lib_suffix 64)
```

Comment:
Shouldn't /lib/ always be a simlink to the correct /lib64 or /lib32 depending on the system?

Are there any systems where this is not true?